README.adoc

```diff
@@ -557,7 +486,7 @@ Next, build the new version of the container image as `2.0-SNAPSHOT`:
 docker build -t system:2.0-SNAPSHOT system/.
 ```
 
-Since you built a new image, it will have to be pushed to the repository again.
+Since you built a new image, it will have to be pushed to the repository again. On the following `docker` and `kubectl` commands, remember to replace `us.icr.io to` your registry and `[your-namespace]` with the namespace you created earlier in the guide.
```

Comment:
```suggestion
Since you built a new image, you need to push it to the repository again. In the following `docker` and `kubectl` commands, remember to replace `us.icr.io` with your registry and `[your-namespace]` with the namespace you created earlier in the guide.
```